tests/benchmark/bm_batch_iterator.cpp

```diff
@@ -93,20 +138,20 @@ class BM_BatchIterator : public benchmark::Fixture {
 size_t BM_BatchIterator::ref_count = 0;
 BM_BatchIterator *BM_BatchIterator::instance = nullptr;
 
-BENCHMARK_DEFINE_F(BM_BatchIterator, BatchedSearch_BF)(benchmark::State &st) {
+BENCHMARK_DEFINE_F(BM_BatchIterator, BF_FixedBatchSize)(benchmark::State &st) {
 
     size_t batch_size = st.range(0);
-    size_t total_res_count = st.range(1);
+    size_t num_batches = st.range(1);
     size_t iter = 0;
     for (auto _ : st) {
         VecSimBatchIterator *batchIterator =
             VecSimBatchIterator_New(bf_index, (*queries)[iter % n_queries].data(), nullptr);
-        size_t res_num = 0;
+        size_t batches_counter = 0;
```

Comment:
question: do we want to count the time it takes to create the batch iterator?
we can reset it on each state iteration

Reply:
yes because RediSearch is creating the batch iterator in every search...